tests/system/Database/Live/SQLite/AlterTableTest.php

```diff
@@ -277,4 +278,66 @@ protected function createTable(string $tableName = 'foo')
         $this->forge->addForeignKey('key_id', $tableName . '_fk', 'id');
         $this->forge->createTable($tableName);
     }
+
+    public function testGetIndexData()
```

Comment:
This test method has nothing to do with ALTER TABLE.
Please create a new TestCase file.